hudi-io/src/main/java/org/apache/hudi/storage/StoragePathInfo.java

```diff
@@ -35,15 +35,23 @@ public class StoragePathInfo implements Serializable {
   private final StoragePath path;
   private final long length;
   private final boolean isDirectory;
+  // blockReplication and blockSize are kept for backwards compatibility
+  // for Hive query engine
```

Comment:
its not just backwards compat right? Based on your description, it feels like even the core functionality for Hive queries need it?
Is this comment needed/useful

Reply:
The core functionality for Hive queries need both fields and they are also used to construct `FileStatus` instances and used in the file system view.  Removed the comments to avoid confusion.